### PR TITLE
Input & Checkbox & Select done

### PR DESCRIPTION
**Pull request by YinzhuSU**

### CODE REVIEW

File: src/components/CustomCheckbox.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Checkbox, Flex } from "antd";
+import Caption from "../components/Caption";
+
+export const CustomCheckbox = ({ className, onChange, content, ...props }) => {
+  const handleChange = (e) => {
+    console.log(`checked = ${e.target.checked}`);
```
Comment (hanqidS2023) [inline src/components/CustomCheckbox.jsx]: Remove this

**Review by hanqidS2023 — APPROVED** [review]

File: src/routes/StyleGuide.jsx [hunk 2]
```diff
@@ -23,13 +31,18 @@ const StyleGuide = () => {
     setIsModalOpen(false);
   };
 
+  const onChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
```
Comment (congruiyS2023) [inline src/routes/StyleGuide.jsx:35]: What's this?

**Review by hanqidS2023 — APPROVED** [review]

File: src/routes/StyleGuide.jsx [hunk 3]
```diff
@@ -117,6 +127,69 @@ const StyleGuide = () => {
...
+          placeholder="Input with label"
+        />
+
+        <CustomInputWithPassword
+          className="w-80"
+          placeholder="input password"
+          iconRender={(visible) =>
+            visible ? <EyeTwoTone /> : <EyeInvisibleOutlined />
+          }
+        />
```
Comment (congruiyS2023) [inline src/routes/StyleGuide.jsx:156]: Please use font specified in the style guide

File: src/routes/StyleGuide.jsx [hunk 4]
```diff
@@ -117,6 +127,69 @@ const StyleGuide = () => {
...
+        <CustomCheckbox className="w-60" content="CheckBox Item" />
+      </Flex>
+
+      <Divider />
+
+      <Flex justify="flex-start" className="px-12">
+        <Heading level={2}>Select</Heading>
+      </Flex>
+
+      <Flex vertical justify="flex-start" gap="middle" className="px-12">
```
Comment (congruiyS2023) [inline src/routes/StyleGuide.jsx:177]: Make text left align as in the style guide

**Review by congruiyS2023 — CHANGES_REQUESTED** [review]

File: src/routes/StyleGuide.jsx (hunk 2 above)
Comment (YinzhuSU) [reply src/routes/StyleGuide.jsx:35]: Placeholder from AntD, will remove in the next PR, wait for 10 mins

File: src/routes/StyleGuide.jsx (hunk 3 above)
Comment (YinzhuSU) [reply src/routes/StyleGuide.jsx:156]: @congruiyS2023 
I don't understand.
This is an icon from AntD. It does not have font.
Also, I did follow the font specified in the style guide in this PR.

File: src/routes/StyleGuide.jsx (hunk 4 above)
Comment (YinzhuSU) [reply src/routes/StyleGuide.jsx:177]: > Make text left align as in the style guide

Changed. Will reflect in the next PR, wait for 10 mins

File: src/routes/StyleGuide.jsx (hunk 2 above)
Comment (congruiyS2023) [reply src/routes/StyleGuide.jsx:35]: Next time you could copy the js version from antd

File: src/routes/StyleGuide.jsx (hunk 3 above)
Comment (congruiyS2023) [reply src/routes/StyleGuide.jsx:156]: Click the file name to see full lines of codes, I'm not referring to the icon here. Also, I don't think this is Barlow: 
image
This is what Barlow should look like: 
image
Notice the bottom right, the font family is not Barlow: 
image

File: src/routes/StyleGuide.jsx (hunk 2 above)
Comment (yinzhusS2023) [reply src/routes/StyleGuide.jsx:35]: Indeed

File: src/routes/StyleGuide.jsx (hunk 3 above)
Comment (hanqidS2023) [reply src/routes/StyleGuide.jsx:156]: just add font-paragraph for Barlow and font-heading for epilogue, it's defined in tailwind.config.js

Comment (congruiyS2023) [reply src/routes/StyleGuide.jsx:156]: Resolved

**Pull request merged**